tests/sigil.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+Test sigil functions
+--SKIPIF--
+<?php if (!extension_loaded("parle")) print "skip"; ?>
+--FILE--
+<?php 
+
+use Parle\Parser;
+use Parle\Lexer;
+use Parle\Token;
+use Parle\SigilToken;
+
+$p = new Parser;
+$p->push("start", "'a' B");
+$p->push("B", "'b'");
+$p->build();
+
+$lex = new Lexer;
+$lex->push("a", $p->tokenId("'a'"));
+$lex->push("b", $p->tokenId("'b'"));
+$lex->push("\\s+", Token::SKIP);
+$lex->build();
+
+$p->consume("a b", $lex);
+
+while (Parser::ACTION_ERROR != $p->action && Parser::ACTION_ACCEPT != $p->action) {
+	switch ($p->action) {
+		case Parser::ACTION_REDUCE:
+			echo $p->sigilInfo(0)->name . "\n";
+			echo $p->sigilInfo(0)->token . "\n";
```

Comment:
The two properties looks pretty much like `Parle\Token` :), please see here https://www.php.net/manual/en/class.parle-token.php . It seems ok for now, but perhaps could think to recompose this to a token object. It'd be still interesting to conceptualize what else could be there in the sigil info.

Thanks

Reply:
A rule rhs element could be a terminal or non-terminal though. Also the request was for a bool for whether a terminal (token) or not.

Reply:
Yeah, with regard to request this fulfills it probably. Perhaps it's just that we can make it in a form doesn't match 1:1, say if a more unified solution would be possible. Perhaps needs some time to digest, as for me.

Thansk